handler/EventHandler_test.go

```diff
@@ -0,0 +1,199 @@
+package handler
```

Comment:
When you rename the impl file as @eldeal requested please update this test file to match + "_test"